products/amazon-aurora-postgresql.md

```diff
@@ -0,0 +1,134 @@
+    latest: "13.11"
+    latestReleaseDate: 2023-05-25
+
+-   releaseCycle: "12.9"
+    releaseDate: 2020-02-25
+    eol: 2025-02-28
+    latest: "12.15"
+    latestReleaseDate: 2023-05-25
+    lts: true
+
+-   releaseCycle: "12"
+    releaseDate: 2020-03-31
+    eol: 2025-02-28
+    latest: "12.15"
+    latestReleaseDate: 2023-05-25
+
+-   releaseCycle: "11"
+    releaseDate: 2019-03-13
+    latest: "11.20"
+    latestReleaseDate: 2023-05-25
+    eol: 2024-01-31
+
+-   releaseCycle: "10.21"
+    releaseDate: 2018-02-28
+    eol: 2022-08-01
+    latest: "10.23"
+    latestReleaseDate: 2023-01-24
+    deprecated: true
+
+-   releaseCycle: "10.20"
+    releaseDate: 2018-02-28
+    eol: 2022-08-01
+    latest: "10.23"
+    latestReleaseDate: 2023-01-24
+    deprecated: true
+
+-   releaseCycle: "10.19"
+    releaseDate: 2018-02-28
+    eol: 2022-08-01
+    latest: "10.23"
+    latestReleaseDate: 2023-01-24
+    deprecated: true
+
+-   releaseCycle: "10.18"
+    releaseDate: 2018-02-28
+    eol: 2022-08-01
+    latest: "10.23"
+    latestReleaseDate: 2023-01-24
+    deprecated: true
+
+-   releaseCycle: "10"
+    releaseDate: 2018-02-27
+    eol: 2023-04-17
+    latest: "10.23"
+    latestReleaseDate: 2023-01-24
+    deprecated: true
+
+-   releaseCycle: "9.6"
+    releaseDate: 2016-11-11
+    eol: 2022-04-30
+    latest: "9.6.24"
+    deprecated: true
+
+---
+
+> [Amazon Aurora for PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.AuroraPostgreSQL.html) is a fully managed, PostgreSQL–compatible,
+> and ACID–compliant relational database engine that combines the speed, reliability, and manageability of Amazon Aurora with the simplicity and cost-effectiveness
+> of open-source databases and is a drop-in replacement for PostgreSQL.
+
+
+## Versioning and Compatibility
+Amazon Aurora reuses code and maintains compatibility with the underlying MySQL and PostgreSQL DB engines.
+However, Aurora has its own version numbers, release cycle, time line for version deprecation, and so on.
+
+## How long Amazon Aurora major versions remain available
+Amazon Aurora major versions remain available at least until community end of life for the corresponding community version.
+
+## How long Amazon Aurora minor versions remain available
+Amazon intends to make each Amazon Aurora minor version of a particular major version available for at least 12 months. At the end of this period, Aurora might apply an auto minor version upgrade to the subsequent default minor version. Such an upgrade is started during the scheduled maintenance window for any cluster that is still running the older minor version.
+
+## Long-term support for selected Amazon Aurora minor versions++
+For each Aurora major version, certain minor versions are designated as long-term-support (LTS) versions and made available for at least three years. That is, at least one minor version per major version is made available for longer than the typical 12 months.
```

Comment:
If we choose to track minor version those must be marked with the `lts: true` attribute.